prototypes/Eclipse GUI Prototype/.project

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@m1cm1c: I’m pretty sure that’s a problem with your setup, because here is the `.project` file.